<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>timeLog</class>
 <widget class="QMainWindow" name="timeLog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1180</width>
    <height>665</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="sideBar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>319</width>
      <height>665</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#sideBar{
	background-color: qlineargradient(spread:pad, x1:0.508, y1:0.0340909, x2:0.503051, y2:1, stop:0 rgba(33, 44, 89, 255), stop:0.497175 rgba(52, 66, 115, 255), stop:0.847458 rgba(72, 87, 142, 255))
}

#lblFilter, #fromLabel, #toLabel{
color: white;
}

#btnFilter{
background-color: white;
border-radius: 5px;
}


QCalendarWidget {
    background-color: white; /* Background color of the calendar */
    border: 1px solid #ccc; /* Border around the calendar */
    selection-background-color: #3498db; /* Background color of selected dates */
    selection-color: white; /* Text color of selected dates */
}

QCalendarWidget QToolButton {
    height: 30px; /* Height of navigation buttons */
    width: 70px; /* Width of navigation buttons */
    color: black; /* Text color of navigation buttons */
    font-size: 14px; /* Font size of navigation buttons */
    font-family: &quot;Poppins SemiBold&quot;; /* Font family of navigation buttons */
    background-color: transparent; /* Background color of navigation buttons */
    border: none; /* Remove button borders */
    padding: 5px; /* Padding inside buttons */
}

QCalendarWidget QAbstractItemView QWidget {
    color: black; /* Text color of the date text */
    font-size: 12px; /* Font size of the date text */
    font-family: &quot;Poppins SemiBold&quot;; /* Font family of the date text */
    background-color: transparent; /* Transparent background */
    border: none; /* No border */
}

QCalendarWidget QAbstractItemView:disabled {
    color: #808080; /* Disabled text color */
}

</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="lblFilter">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>281</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Poppins</family>
       <pointsize>30</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>DATE FILTER</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="fromLabel">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>280</y>
       <width>71</width>
       <height>34</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Poppins</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>From</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateStart">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>280</y>
       <width>126</width>
       <height>34</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Poppins</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: black;</string>
     </property>
     <property name="dateTime">
      <datetime>
       <hour>0</hour>
       <minute>0</minute>
       <second>0</second>
       <year>2024</year>
       <month>1</month>
       <day>1</day>
      </datetime>
     </property>
     <property name="displayFormat">
      <string>yyyy-MM-dd</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="toLabel">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>340</y>
       <width>71</width>
       <height>34</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Poppins</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>To</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateEnd">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>340</y>
       <width>126</width>
       <height>34</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Poppins</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: black;</string>
     </property>
     <property name="dateTime">
      <datetime>
       <hour>0</hour>
       <minute>0</minute>
       <second>0</second>
       <year>2024</year>
       <month>1</month>
       <day>1</day>
      </datetime>
     </property>
     <property name="displayFormat">
      <string>yyyy-MM-dd</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btnFilter">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>400</y>
       <width>220</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Poppins</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Filter</string>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="btnCard">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>450</y>
       <width>220</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Poppins</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(85, 255, 127);
border-radius: 5px;</string>
     </property>
     <property name="text">
      <string>Create Time Card</string>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QTableWidget" name="employeeListTable">
    <property name="geometry">
     <rect>
      <x>318</x>
      <y>0</y>
      <width>860</width>
      <height>665</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Poppins SemiBold</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">/* Header Design */

QHeaderView::section {
	font-family: Poppins;
	font-weight: bold;
	font-size: 13px;
	color: white;
	background-color: &quot;#344273&quot;;
	border: none;
}

/* Style for the selected row */
QTableView::item:selected:active {
	 selection-background-color: &quot;#9DADDF&quot;;
}

/* Style for the alternate row */
QTableWidget {
	alternate-background-color: &quot;#DDDDDD&quot;;
}</string>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustIgnored</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="tabKeyNavigation">
     <bool>false</bool>
    </property>
    <property name="showDropIndicator" stdset="0">
     <bool>false</bool>
    </property>
    <property name="dragDropOverwriteMode">
     <bool>false</bool>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="textElideMode">
     <enum>Qt::ElideRight</enum>
    </property>
    <property name="showGrid">
     <bool>false</bool>
    </property>
    <property name="gridStyle">
     <enum>Qt::SolidLine</enum>
    </property>
    <property name="cornerButtonEnabled">
     <bool>false</bool>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>50</number>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>170</number>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>Bio No.</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>TransDate</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Time</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>MachCode</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Sched</string>
     </property>
    </column>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
